src/puppetlabs/puppetdb/scf/storage.clj

```diff
@@ -1404,7 +1446,7 @@
                          [sql-timestamp certname sql-timestamp])))))
 
 (pls/defn-validated expire-stale-nodes [horizon :- Period]
-  "Expires nodes with no activity within the provided horizon (prior
+  "Expires nodes with no activity within the provided horizon (prior/catalog
```

Comment:
Not sure I followed "prior/catalog" -- might be worth elaborating a little more.

Reply:
oops that's a accidental paste from the clipboard, I'll remove that